<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWindow</class>
 <widget class="QWidget" name="SearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>588</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Поиск</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QToolButton" name="btnCatalogsSelect">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/catalog.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnBooksSelect">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/book.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtSearch">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnFind">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/search_1.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnFindChapters">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/labels.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextBrowser" name="edtText">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Источник:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtSource">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnFont">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../bin/images.qrc">
         <normaloff>:/images/font.png</normaloff>:/images/font.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="lstResults">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>161</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>161</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lstText">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>161</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/search_1.png</normaloff>:/images/search_1.png</iconset>
   </property>
   <property name="text">
    <string>Поиск по текстам</string>
   </property>
   <property name="toolTip">
    <string>Поиск по текстам</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionFindChapters">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/labels.png</normaloff>:/images/labels.png</iconset>
   </property>
   <property name="text">
    <string>Поиск по заголовкам</string>
   </property>
   <property name="toolTip">
    <string>Поиск по заголовкам</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_books_select">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/book.png</normaloff>:/images/book.png</iconset>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="toolTip">
    <string>Выбор книг для поиска</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/font.png</normaloff>:/images/font.png</iconset>
   </property>
   <property name="text">
    <string>Выбор шрифта</string>
   </property>
   <property name="toolTip">
    <string>Выбор шрифта</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_catalogs_select">
   <property name="icon">
    <iconset resource="../../bin/images.qrc">
     <normaloff>:/images/catalog.png</normaloff>:/images/catalog.png</iconset>
   </property>
   <property name="text">
    <string>Выбор каталогов для поиска</string>
   </property>
   <property name="toolTip">
    <string>Выбор каталогов для поиска</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../bin/images.qrc"/>
 </resources>
 <connections/>
</ui>
